metadata/src/test/java/org/apache/kafka/image/ClusterImageTest.java

```diff
@@ -186,6 +264,19 @@ public void testImage2RoundTrip() {
         testToImage(IMAGE2);
     }
 
+    public void testApplyDelta2() {
```

Comment:
Is this missing the `@Test` annotation?

Reply:
Sorry I should have added that after enabling that test, it is failing.
Expected:
`brokers([...], 1(BrokerRegistration(id=1, epoch=1001, incarnationId=U52uRe20RsGI0RvpcTx33Q, listeners=[Endpoint(listenerName='PLAINTEXT', securityProtocol=PLAINTEXT, host='localhost', port=9093)], supportedFeatures={foo: 1-3}, rack=Optional.empty, fenced=true, inControlledShutdown=true, isMigratingZkBroker=false, directories=[])), [...]`
Actual:
`brokers([...], 1(BrokerRegistration(id=1, epoch=1001, incarnationId=U52uRe20RsGI0RvpcTx33Q, listeners=[Endpoint(listenerName='PLAINTEXT', securityProtocol=PLAINTEXT, host='localhost', port=9093)], supportedFeatures={foo: 1-3}, rack=Optional.empty, fenced=false, inControlledShutdown=false, isMigratingZkBroker=false, directories=[])), [...]`
Broker 1 has `fenced=false` and `inControlledShutdown=false` while we expected both to be `true`.